Packages/com.unity.inputsystem/InputSystem/Controls/InputControl.cs

```diff
@@ -991,17 +1022,73 @@ internal virtual IEnumerable<object> GetProcessors()
     public abstract class InputControl<TValue> : InputControl
         where TValue : struct
     {
+        /// <summary>
+        /// Returns the current value of the control after processors have been applied.
+        /// </summary>
+        /// <remarks>
+        /// Note that this property implements caching to avoid applying processors when the underlying
+        /// control has not changed. With this in mind, be aware of processors that use global state, such
+        /// as the <see cref="Processors.AxisDeadzoneProcessor"/>. Unless the control has been actuated, the processors
```

Comment:
Feels like we would benefit from a diagnostic mode where we always run full value read from with-in value and compare it to cached value, that way we would catch situations like changing globals in processors and make it easier to diagnose them